test/cpp/security/tls_certificate_verifier_test.cc

```diff
@@ -161,6 +161,85 @@ TEST(TlsCertificateVerifierTest,
   EXPECT_EQ(sync_status.error_message(), "Hostname Verification Check failed.");
 }
 
+TEST(TlsCertificateVerifierTest,
+     NoOpCertificateVerifierSucceedsWithVerifiedRootCertSubject) {
+  grpc_tls_custom_verification_check_request request;
+  memset(&request, 0, sizeof(request));
```

Comment:
nit: Why is this line necessary? I'm guessing this is copied from existing code, but I think this line is NO-OP.

Similarly below.

Reply:
Interestingly, the unset test`NoOpCertificateVerifierSucceedsWithoutVerifiedRootCertSubject` fails without this line.  I presume this clears the memory, and perhaps without it we are accessing garbage?

Reply:
Once this PR is submitted, we should investigate this. I'm worried there's some order-dependence or shared state to these tests.